include/odp/api/spec/traffic_mngr.h

```diff
@@ -605,6 +605,36 @@ odp_tm_t odp_tm_find(const char            *name,
  */
 int odp_tm_capability(odp_tm_t odp_tm, odp_tm_capabilities_t *capabilities);
 
+/** Start a TM system.
+ *
+ * odp_tm_start() needs to be used to start an already created / found TM system
+ * By default, all the TM systems are in stopped state.
+ * Calling this function on an already started TM system will be ignored.
```

Comment:
Do we need to allow this? Why not "This function may not be called for a TM system that is already in the started state."?